backend/src/main/java/gov/cdc/usds/simplereport/db/model/PhoneNumber.java

```diff
@@ -0,0 +1,102 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.google.i18n.phonenumbers.NumberParseException;
+import com.google.i18n.phonenumbers.PhoneNumberUtil;
+import gov.cdc.usds.simplereport.db.model.auxiliary.PhoneType;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import org.hibernate.annotations.Type;
+
+@Entity
+public class PhoneNumber extends AuditedEntity {
+  @ManyToOne(optional = false)
+  @JoinColumn(name = "person_internal_id")
+  private Person person;
+
+  @Enumerated(EnumType.STRING)
+  @Type(type = "pg_enum")
+  @Column
+  private PhoneType type;
+
+  @Column private String number;
+
+  public PhoneNumber() {
+    /* For Hibernate */
+  }
+
+  public PhoneNumber(PhoneType type, String number) {
+    this.type = type;
+    this.number = number;
+  }
+
+  public PhoneNumber(Person person, PhoneType type, String number) {
```

Comment:
i think we need to hit the brakes on the assumption that each phone number is associated with one person. as we roll out to more diverse communities that assumption will fall apart fast.